lib/scrum/card_type_detection.rb

```diff
@@ -1,5 +1,9 @@
 module Scrum
   module CardTypeDetection
+    def sticky?(card)
+      card.labels.any? { |l| l.name == "Sticky" }
```

Comment:
Would be great if I'd  be able to use it, but `include?` won't look at the `name` property.